src/Resources/PageResource.php

```diff
@@ -356,10 +367,6 @@ public static function relatablePages(
         $model = $request->findModelOrFail($request->resourceId);
         $query->where('id', '!=', $model->id);
 
-        if ($field->attribute === 'parent') {
```

Comment:
@harmenjanssen I think I have added this for the 'parent' field, so you could only choose a parent page with the same language. You sure we don't need this?

Reply:
No, because this function turned out to not be used for that relation at all. 🙂 

We're using `getPageOptionsForSelect` for that: https://github.com/grrr-amsterdam/nova-pages-tool#rendering-a-page-selector-elsewhere-in-your-cms

Reply:
Although that function could use a language parameter for filtering.

Reply:
Will add that feature separately.